lib/src/presentation/authenticator_widget.dart

```diff
@@ -70,15 +70,17 @@ class _AuthenticatorWidgetState extends State<AuthenticatorWidget> {
   late final StreamSubscription lockSubscription;
   OverlayEntry? overlayEntry;
   bool _isShowingSplashScreen = true;
+  late final Stream<LockState> lockState ;
 
   @override
   void initState() {
     super.initState();
+    lockState = widget.authenticator.lockState;
```

Comment:
This helps to ensure the method `_checkInitialLockStatus` of `lockState` not called twice, as we need it in line 125.